src/Cxx/Geovis/GeoGraticle.md

```diff
@@ -1,7 +1,7 @@
 ### Description
 
-vtkGeoGraticle generates polydata to illustrate the distortions introduced by a map projection. The level parameter specifies the number of lines to be drawn. Poles are treated differently than other regions; hence the use of a Level parameter instead of a NumberOfLines parameter. The latitude and longitude are specified as half-open intervals with units of degrees. By default the latitude bounds are [-90,90[ and the longitude
-bounds are [0,180[.
+vtkGeoGraticle generates polydata to illustrate the distortions introduced by a map projection. The level parameter specifies the number of lines to be drawn. Poles are treated differently than other regions; hence the use of a Level parameter instead of a NumberOfLines parameter. The latitude and longitude are specified as half-open intervals with units of degrees. By default the latitude bounds are [-90,90] and the longitude
+bounds are [0,180].
 
```

Comment:
Could you check this? Given that these are half-open intervals I think the they should be [-90...90) and [0...180) or maybe the open interval (-90...90) -  more likely. In looking at the code I think these should be (-90...90) and (-180...180) i.e. open intervals.

Reply:
@ajpmaclean Sure I will change it.